solidity/test/DepositUtilsTest.js

```diff
@@ -641,6 +642,71 @@ describe("DepositUtils", async function() {
       )
     })
   })
+  describe("enableWithdrawal()", async () => {
+    beforeEach(async () => {
+      await createSnapshot()
+    })
+
+    afterEach(async () => {
+      await restoreSnapshot()
+    })
+
+    it("correctly adds value to withdrawalAllowances", async () => {
+      const value = new BN(10000)
+      const beneficiary = accounts[1]
+
+      await testDeposit.enableWithdrawal(accounts[1], value)
+
+      const withdrawable = await testDeposit.getWithdrawAllowance.call({
+        from: beneficiary,
+      })
+
+      expect(withdrawable).to.eq.BN(value)
+    })
+  })
+
+  describe("withdrawFunds()", async () => {
+    beforeEach(async () => {
+      await createSnapshot()
+    })
+
+    afterEach(async () => {
+      await restoreSnapshot()
+    })
+
+    it("Correctly withdraws withdrawable balance at end-state", async () => {
```

Comment:
We should make this test check every state that is an end-state, and make the next test check every state that isn't.

Also, as far as I know we don't capitalize our “it” strings. You should consider these the end of a sentence that follows from previous `describe`s (e.g. `DepositUtils withdrawFunds() correctly withdraws withdrawable balance at end-state`).